standard/expressions.md

```diff
@@ -3434,8 +3434,16 @@ The predefined subtraction operators are listed below. The operators all subtrac
   ```csharp
   D operator –(D x, D y);
   ```
-  If the first operand is `null`, the result of the operation is `null`. Otherwise, if the second operand is `null`, then the result of the operation is the value of the first operand. Otherwise, both operands represent invocation lists ([§20.2](delegates.md#202-delegate-declarations)) having one or more entries, and the result is a new invocation list consisting of the first operand's list with the second operand's entries removed from it, provided the second operand’s list is a proper contiguous sublist of the first's. (To determine sublist equality, corresponding entries are compared as for the delegate equality operator ([§12.11.9](expressions.md#12119-delegate-equality-operators)).) Otherwise, the result is the value of the left operand. Neither of the operands' lists is changed in the process. If the second operand's list matches multiple sublists of contiguous entries in the first operand's list, the right-most matching sublist of contiguous entries is removed. If removal results in an empty list, the result is `null`.
-  
+  The semantics are as follows:
+  - If the first operand is `null`, the result of the operation is `null`.
+  - Otherwise, if the second operand is `null`, then the result of the operation is the value of the first operand.
+  - Otherwise, both operands represent invocation lists ([§20.2](delegates.md#202-delegate-declarations)).
```

Comment:
I think it's worth making this "non-empty invocation lists" (as a slightly simpler way of writing the "invocation lists having one or more entries" from the origina text).